src/layout/navigation/Navigation.tsx

```diff
@@ -36,7 +36,7 @@ const Navigation: React.FC<NavigationProps> = ({
           mx={{ default: '0', sm: '0', md: '40px', lg: '75px' }}
         >
           <FlexColumn width={1}>
-            <LogoWrapper to="/">
+            <LogoWrapper onClick={() => setOpen(!isOpen)} to="/">
```

Comment:
Double-check the logic here 😜 if you click on the logo when the menu is closed, observe the behaviour!